rebench/model/measurement.py

```diff
@@ -58,6 +58,10 @@ def from_str_list(cls, data_store, str_list, line_number=None, filename=None):
 
         return Measurement(invocation, iteration, value, unit, run_id,
                            criterion, line_number, filename)
+    @classmethod
+    def get_column_headers(cls):
+        return["Invocation","iteration","value","Unit","Criterion","Benchmark",
+               "Executor","Suite","Cores","Input size","Var value","Machine"]
```

Comment:
Could you please change this so that it uses the same logic as `from_str_list`, in the sense that it relies on the other classes to determine the names of the columns?
This way, each class is responsible for the same set of columns when serializing and deserializing data, and we don't shift the complete burden to Measurement.
This will also help keeping things correctly in sync, because the columns are defined next to the code that implements them.

Please also decide on a consistent way of capitalizing the names. We may also want to avoid spaces in names and use [camelCase](https://en.wikipedia.org/wiki/Camel_case) instead to avoid issues with parsing in CSV libraries that may get confused.